tests/_test_fit.py

```diff
@@ -8,8 +8,8 @@
 import os
 import sys
 
-from hmf import MassFunction
-from hmf import fit
+from src.hmf import MassFunction
```

Comment:
This completely defeats the purpose of having src-layout in the repo. Here, read some explanation: https://blog.ganssle.io/articles/2019/08/test-as-installed.html / https://hynek.me/articles/testing-packaging/.